backends/platform/ios7/ios7_app_delegate.mm

```diff
@@ -112,11 +112,19 @@ - (void)applicationDidEnterBackground:(UIApplication *)application {
 	[_view saveApplicationState];
 }
 
+#ifdef __IPHONE_13_2
+- (BOOL)application:(UIApplication *)application shouldSaveSecureApplicationState:(NSCoder *)coder {
+#else
 - (BOOL)application:(UIApplication *)application shouldSaveApplicationState:(NSCoder *)coder {
+#endif
```

Comment:
Won't that change break compiling with iOS 13 SDK (or above) but targeting and running on older iOS?
Or is the `__IPHONE_13_2` define related to the deployment target version and not SDK version?

If there is a doubt I can test since my old iPad is on iOS 12.

Reply:
I've tested this on an iPad mini version 1 running iOS 9. I built the project using Xcode 13. I guess it will choose depending on the minimum target version.

Reply:
Did you only checked that it compiles and runs, or also that it works properly?
My suspicion is that when compiling with SDK 13 or above, it compiles with the newer name. The compilation should pass, and the application will run. But they would never be called, which would cause the state to not be saved/restored. I will double check with my device just to be sure.

Reply:
I confirmed that this change breaks saving/restoring the state. Those functions do not get called when running on iOS 12. I also confirmed that using the old functions still works on iOS 16.2. So I will revert this commit.